### PR TITLE
Add Quintessence QRNG

### LINKED ISSUE #1142: Update chain list page to mention ANU and BYOG usage
This is related to previous mentions about which chains ANU and byog can use.

### PR DESCRIPTION
**Pull request by Ashar2shahid**
PR Adds Quintessence QRNG info along with information of what chains it serves

### CODE REVIEW

**Review by wkande — CHANGES_REQUESTED** [review]
@Ashar: I think you are missing a testnet for ANU (Goerli) See the **[PR here](https://github.com/api3dao/api3-docs/pull/1137)** where Burak has indicated that ANU does in fact use Goerli. 

I was thinking to add it myself but worry I might not know the `contract address` and `minConfiguration`.

**Comment by Ashar2shahid** [comment]
this is what was said in the [PR](https://github.com/api3dao/api3-docs/pull/1137#discussion_r1041869086)
> Previously
ANU: all mainnets and goerli
Now
ANU: all mainnets (and no testnets)
byog: all testnets (and no mainnets)           

according to that this is correct

File: docs/qrng/reference/chains.md [hunk 1]
```diff
@@ -66,11 +67,13 @@ to not break as well.
 | <ChainName chainId="30"/>    | 30       | 0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd <CopyIcon text="0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd"/> | 3                  |
 
 </div>
+
 :::
 
-::: tab testnets
+::: tab BYOG
```
Comment (dcroote) [inline docs/qrng/reference/chains.md]: For consistency:
```suggestion
::: tab byog
```

**Review by dcroote — APPROVED** [review]
Only a minor comment

**Review by Ashar2shahid — COMMENTED** [review]
@wkande I think we can go ahead and merge this now

**Comment by wkande** [comment]
Will be merging this today as soon as I duplicate this into the VitePress docs.

**Pull request merged**